src/preprocess.py

```diff
@@ -191,18 +186,27 @@ def _find_cached_file(exp_dir: str, global_exp_cache_dir: str,
 
 def _build_embeddings(args, vocab, emb_file: str):
     ''' Build word embeddings from scratch (as opposed to loading them from a pickle),
-    possibly using a fastText model or precomputed fastText / GloVe embeddings. '''
+    using precomputed fastText / GloVe embeddings. '''
+
+    # Load all the word embeddings based on vocabulary
     log.info("\tBuilding embeddings from scratch")
-    if args.fastText:
-        word_embs, _ = get_fastText_model(vocab, args.d_word,
-                                          model_file=args.fastText_model_file)
-        log.info("\tUsing fastText; no pickling of embeddings.")
-        return word_embs
-
-    word_embs = get_embeddings(vocab, args.word_embs_file, args.d_word)
```

Comment:
This seemed like a nice helper method. Why get rid of it?

Reply:
After getting rid of the optional loading of the fastText model, it was just a wrapper around `get_embeddings` without any extra functionality.